### ISSUE TITLE
Indistinct path renders as solid line when dash point is too close to the end

### ISSUE THREAD

**Issue by genebeveridge** [OpenOrienteering/mapper#2105]
### Steps to reproduce
1. Draw a section of indistinct path with ISSprOM 
2. Change a normal point to a dash point (ctrl + space)
3.

### Actual behaviour
Depending on the position of the dash point, part of the path will become a solid line. Around 8mm long so possibly because the dimension of the end section would be below the lower limits on the dimensions from ISSprOM.

### Expected behaviour
Indistinct path should retain it "double dash" structure instead of becoming a solid line. Perhaps a warning that something is below minimum side can be added somewhere, but I'm confident a solid line is not preferred by the majority of mappers.

### Configuration
Mapper Version: 0.9.5
Operating System: Windows 11 - Version 10.0.22000 Build 22000
![Untitled 2](https://user-images.githubusercontent.com/92624993/197849759-111c88ea-0853-472c-8163-02c8e2e4a684.png)
![Untitled](https://user-images.githubusercontent.com/92624993/197849761-e72bfce1-f87e-46cb-aa1d-06718eeeb80e.png)

**Comment by krticka** [comment]
Using dash point is problematic for short sections of this symbol in Mapper and it is not even perfect in Ocad. What I usually do is that I draw such places using two separated small footpaths (506).

**Comment by dg0yt** [comment]
It is implemented how it should work IMO: It doesn't let map maker get away easily with short lines below minimum size (as OCAD does). But it lets them be explicit on a case-by-case base (i.e. as an exception) by drawing separate lines.

Maybe there could be smarter algorithms for placing the dashes. But to fully benefit from any solution, we need full course setting (for printing) in Mapper.

**Comment by genebeveridge** [comment]
Yes, probably no perfect solution. IMO it would be best to keep the style
of the symbol in tact, and forcing a solid black line without an
explanation is frustrating. Better alternatives include displaying a
one-time notification or changing the symbol to a solid red line.

In the map I have just been given to work on, I was wondering what the
solid black lines were, I've just not realised they are small indistinct
path sections. So the previous mapper was obviously not deterred.

OCAD lets you draw below standard size paths, but haven't played with the
dash point in OCAD.

**Comment by MartinHore** [comment]
Sorry to come to this late. I doubt there is a satisfactory coding solution to this issue (which I encounter quite often). The IOF indistinct path symbol requires four dashes and one longer gap to display it properly.  That will often exceed the length of the path segment concerned.  I create something which looks right in each individual circumstance. Sometimes that will be just replacing 507 with 506 (whether a path is distinct or indistinct is a bit arbitrary anyway). Other times I will use 506 but leave a gap at each end (ie where it joins another path). And sometimes I will just omit the path, especially if it's not distinct at its ends. Martin
